src/renderer/components/dock/logs/controls.tsx

```diff
@@ -44,13 +47,26 @@ const NonInjectedLogControls = observer(({ openSaveFileDialog, model }: Dependen
     model.reloadLogs();
   };
 
+  const downloadAllLogs = async () => {
+    const pod = model.pod.get();
+    
+    if (pod) {
+      const logs = await callForAllLogs(pod, { timestamps: showTimestamps, previous });
+      
+      openSaveFileDialog(`${pod.getName()}.log`, logs, "text/plain");
+    }
+  };
```

Comment:
Wouldn't it be a better idea to have this be part of the model directly?

Reply:
Moved to injectables.